src/bin/nativelink.rs

```diff
@@ -12,11 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
+
```

Comment:
what are these spaces for?

Reply:
I will remove these and make sure formatting gets run again. My mistake on these spaces here.